libpod/define/annotations.go

```diff
@@ -1,5 +1,11 @@
 package define
 
+import (
```

Comment:
This code should not be part of libpod/define. Define is for definitions only, so it remains lightweight and safe to import everywhere. I recommend moving the annotation code into a new package under pkg/.